packages/utils/build-tools/src/transform-selector.ts

```diff
@@ -0,0 +1,10 @@
+import upstreamTransformer = require('metro-react-native-babel-transformer');
+import svgTransformer = require('react-native-svg-transformer');
```

Comment:
This is kind of a strange mix of TS and JS module usage here.  I'm surprised you aren't getting lint/build errors here during the typescript build.  Try changing these to:
```ts
import * as upstreamTransformer from 'metro-react-native-babel-transformer';
import * as svgTransformer from 'react-native-svg-transformer';
```